source-code/website/src/services/markdown/src/tags/Figure.tsx

```diff
@@ -1,7 +1,18 @@
-export function Figure(props: { src: string; alt: string; caption: string }) {
+export function Figure(props: {
+  src: string;
+  alt: string;
+  caption: string;
+  height: string;
```

Comment:
Good but hardcoded width height values break responsiveness. 

Can you please open a follow up PR that removes `width` and `height` again/replaces it with responsive values?

Reply:
@jannesblobel

Reply:
https://github.com/inlang/inlang/pull/414